rootfs/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM quay.io/deis/base:0.3.0
+
+ENV VERSION=0.3.8.linux-amd64.go1.6.2
+# Add nsq user and group
+RUN adduser --system \
+	--shell /bin/bash \
+	--disabled-password \
+	--home /opt/nsq \
+	--group \
+	nsq
+
+RUN curl -Ls https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 > /usr/bin/jq \
+    && chmod +x /usr/bin/jq \
+    && curl -s https://s3.amazonaws.com/bitly-downloads/nsq/nsq-$VERSION.tar.gz | tar xz \
+    && mv /nsq-$VERSION/bin/* /bin \
+    && rm -r /nsq-$VERSION
+
+COPY . /
+RUN chown -R nsq:nsq /opt/nsq
+# USER nsq
```

Comment:
It's a good practice to run things in containers as a non-root user in order to limit what a malicious party can do to the host's kernel (which it shares with the container).  In the case of nsq, this should be pretty easily accomplished, _I think_, because it seems nsqd is already listening only on unprivileged ports and since we're not persisting queues, I don't think there are any permission considerations that would require us to run as `root` either. Can we uncomment this line?